HARK/abstract/tests/consindshk.yml

```diff
@@ -0,0 +1,39 @@
+---
+states: !StateSpace
+  variables:
+    - !State
+      name: m
+      short_name: money
+      long_name: market resources
+      latex_repr: \mNrm
+    - !State
+      name: &name stigma
+      short_name: &short_name risky share
+      long_name: &long_name risky share of portfolio
+      latex_repr: &latex_repr \stigma
+
+actions: !ActionSpace
+  variables:
+    - !Action
+      name: c
+      short_name: consumption
+      long_name: consumption
+      latex_repr: \cNrm
+    - !Action
+      name: *name
+      short_name: *short_name
+      long_name: *long_name
+      latex_repr: *latex_repr
+
+post_states: !PostStateSpace
+  variables:
+    - !PostState
+      name: a
```

Comment:
Looks like you've repeated the post_states block twice in this file?

Not sure how @mnwhite feels, but maybe we don't need to draw a firm distinction between states and post states like this.

Or the labels could be inside the variable, not part of the document structure.

Compare:
```
var_type_1:
   variables:
       - !VarTypeClass1
           details

var_type_2:
   variables:
       - !VarTypeClass2
           details
```

with

```
variables:
    - !VarTypeClass1
       details
    - !VarTypeClass2   
       details
```

The latter is same information, but fewer lines.